base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp

```diff
@@ -115,29 +116,45 @@ bool PotentiallyHostArtifactsPath(const std::string& host_artifacts_path) {
   return !result.empty();
 }
 
-Result<void> UpdateWebrtcDeviceId(cvd_common::Args& args,
-                                  const selector::LocalInstanceGroup& group) {
+Result<std::vector<std::string>> ExtractWebRTCDeviceIds(
+    cvd_common::Args& args) {
   std::string flag_value;
   std::vector<Flag> webrtc_device_id_flag{
       GflagsCompatFlag("webrtc_device_id", flag_value)};
-  // ConsumeFlags modifies args, a copy is needed because only the presence
-  // of the flag is being tested.
-  std::vector<std::string> copied_args{args};
-  CF_EXPECT(ConsumeFlags(webrtc_device_id_flag, copied_args));
+  CF_EXPECT(ConsumeFlags(webrtc_device_id_flag, args));
```

Comment:
Because `args` is passed down as a mutable reference all the way from `CvdStartCommandHandler::Handle`, this will consume the `webrtc_device_id` flag such that it does not actually make it to `cvd_internal_start`. Is that intended, or should the flag be preserved for `cvd_internal_start`?

Reply:
It's intended, the flag is re-added in line 153 with this value or the newly generated one.